packages/wonder-stuff-i18n/src/utils/pofile-utils.js

```diff
@@ -0,0 +1,170 @@
+// @flow
+import fs from "fs";
+import path from "path";
+
+import fglob from "fast-glob";
+import PO from "pofile";
+
+import {extractStrings} from "./extract-i18n.js";
+import {getIgnoreGlobs} from "./i18n-utils.js";
+
+import type {ExtractedString} from "./extract-i18n.js";
+
+type POItem = {|
+    msgid: string,
+    msgid_plural: string,
+    flags: {[string]: boolean},
+    references: Array<string>,
+    extractedComments: Array<string>,
+|};
```

Comment:
If you wanted to be fancy you could create a libdef for `pofile` in flow-typed.  What you have it totally fine as well.  No action req'd.